examples/minimal-mdns/BUILD.gn

```diff
@@ -67,7 +67,7 @@ executable("mdns-advertiser") {
   deps = [
     ":minimal-mdns-example-common",
     "${chip_root}/src/lib",
-    "${chip_root}/src/lib/mdns",
+    "${chip_root}/src/lib/mdns:mdns",
```

Comment:
is `:mdns` not the default? could we skip this line change here and below?

Reply:
As it turns out, yes we can. Thanks!